frontend/src/layouts/search-layout.vue

```diff
@@ -54,6 +55,14 @@ watch(mainPageY, (y) => {
 
 onMounted(() => {
   mainPageElement.value = document.getElementById("main-page")
+  headerElement.value = document.getElementsByClassName(
+    "header-el"
+  )[0] as HTMLElement
+})
+
+const { height } = useElementSize(headerElement)
+watch(height, (height) => {
+  uiStore.setHeaderHeight(height)
```

Comment:
This is miserable :disappointed: I wish there was another easy way to fix this, but we've put ourselves into such a hole by managing the layout of these elements with JavaScript instead of CSS :face_with_diagonal_mouth: 

At the very least, it would be nice to clean this up in a fast-follow by creating a component to encapsulate this behaviour instead of selecting on the classname in a random part of the app. 

Not blocking this due to criticality.